backend/reservations/views/rooms.py

```diff
@@ -208,32 +205,32 @@ def swap_request(request):
         swap_req = Room.objects.filter(number=room_num)
         try:
             swap_req_email = swap_req[0].guest.email
-            print(f'[+] Swap request sent to: {swap_req_email}')
+            logger.info(f'[+] Swap request sent to: {swap_req_email}')
         except KeyError as e:
             return Response("No email found for that room", status=status.HTTP_400_BAD_REQUEST)
 
         logger.info(f"[+] Sending swap req from {requester_email} to {swap_req_email}")
 
         body_text = f"""
-        
-Someone would like to trade rooms with you for Room Service. Since rooms are randomly assigned, we built this tool for everyone to trade rooms and get the placement they want. If you are open to trading rooms, contact this person via the info below. Sort out the details, then one of you will generate a swap code in Roombaht and send it to the other. Enter the code, switch-aroo magic happens, and you check-in as normal. 
+
+Someone would like to trade rooms with you for Room Service. Since rooms are randomly assigned, we built this tool for everyone to trade rooms and get the placement they want. If you are open to trading rooms, contact this person via the info below. Sort out the details, then one of you will generate a swap code in Roombaht and send it to the other. Enter the code, switch-aroo magic happens, and you check-in as normal.
 
 Contact info: {msg}
 
-After you have contacted the person asking to trade rooms with you, click this link to create the swap code and trade rooms: http://ec2-3-21-92-196.us-east-2.compute.amazonaws.com:3000/rooms 
+After you have contacted the person asking to trade rooms with you, click this link to create the swap code and trade rooms: http://ec2-3-21-92-196.us-east-2.compute.amazonaws.com:3000/rooms
```

Comment:
`http://ec2-3-21-92-196.us-east-2.compute.amazonaws.com/rooms` -> `http://rooms.take3presents.com/rooms`
Would this be more accurate? Do we need to include the port if we have the DNS redirect?
Can address this in another PR